### PR TITLE
EN6-159 fixed update content

### PR DESCRIPTION
**Pull request by ffleandro**

### CODE REVIEW

**Comment by entando-service** [comment]
Something went wrong. Investigate!

**Comment by ffleandro** [comment]
Retest this please

**Comment by entando-service** [comment]
Something went wrong. Investigate!

**Comment by ffleandro** [comment]
Retest this please

**Comment by entando-service** [comment]
Something went wrong. Investigate!

**Comment by entando-service** [comment]
All is Well

File: plugins/entando-plugin-jacms/src/main/java/com/agiletec/plugins/jacms/aps/system/services/content/model/Content.java [hunk 1]
```diff
@@ -168,7 +169,7 @@ public void setDefaultModel(String defaultModel) {
     @Override
     public IApsEntity getEntityPrototype() {
         Content content = (Content) super.getEntityPrototype();
-        content.setStatus(STATUS_NEW);
+        content.setStatus(STATUS_DRAFT);
```
Comment (eugeniosant) [inline plugins/entando-plugin-jacms/src/main/java/com/agiletec/plugins/jacms/aps/system/services/content/model/Content.java]: The status of the prototype has to be "new", draft is the first status that the editor can apply to a content

**Comment by entando-service** [comment]
Something went wrong. Investigate!

**Comment by eugeniosant** [comment]
retest this please

**Comment by entando-service** [comment]
All is Well

**Pull request merged**